CONTRIBUTING.md

```diff
@@ -1,223 +1,3 @@
-
-```
-PRIVACY_REQUEST = "/privacy-request"
-PRIVACY_REQUEST_DETAIL = "/privacy-request/{privacy_request_id}"
-```
-
-would both resolve as `/api/v1/privacy-request` and `/api/v1/privacy-request/{privacy_request_id}` respectively.
-
-
-## PR Merging Guidelines
-
-Pull requests should be submitted with a clear description of the issue being handled, including links to any external specifications, JIRA tickets or Github issues. PRs should not be merged by the person submitting them, except in rare and urgent circumstances.
-
-
-## Linting -- Pylint
-
-Linting is done using the Pylint library. Full docs can be found here: https://docs.pylint.org/en/1.6.0/tutorial.html. Our Pylint rules are configured inside `pyproject.toml`.
-
-- Run project linting with: `make pylint`
-
-
-## Autoformatting -- Black
-
-Black is used to autoformat the repo's code. Full docs can be found here: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html. Our Black rules are configured inside `pyproject.toml`/
-
-- Run autoformatting with: `make black`
-
-
-## Type checking -- Mypy
-
-Mypy is used to enforce static typechecking at compile time. Full docs can be found here: https://mypy.readthedocs.io/en/stable/config_file.html. Our Mypy rules are configured inside `mypy.ini`.
-
-- Run type checking with: `make mypy`
-
-
-NB. pre-commit checks can be run with `make check-all`.
-
-
-## General debugging -- pdb
-
-The project uses `pdb` for debugging as a `dev-requirement`. You can set breakpoints with `pdb` in much the same way you'd set them using `debugger` in Javascript. Insert `import pdb; pdb.set_trace()` into the line where you want the breakpoint to set, then run your Python code.
-
-
-## Docker
-
-The project is based on Docker Compose. To further inspect the commands we generally use you can read `Makefile` in the root of this repo.
-
-### Clearing local Docker instances
-
-Occasionally when developing you'll run into issues where it's beneficial to remove all existing Docker instances in order to recreate them based on some updated spec. Some commands to do this are below:
-
-  - Stop all running containers: `docker-compose down`
-  - Delete all local containers: `docker rm -f $(docker ps -a -q)`
-  - Delete all local Docker volumes: `docker volume rm $(docker volume ls -q)`
-  - Remove temp. files and installed dependencies: `make clean`
-  - Delete all stopped containers, all networks not used by a container, all dangling images, and all build cache: `docker system prune`
-  - Recreate the project: `make compose-build`
+The Fides project, which includes Fideslang, Fidesops, and Fidesctl, adheres to the following [Contribution Guidelines](https://ethyca.github.io/fidesops/development/overview/).
```

Comment:
Similar to above i might rephrase to something like Contributions welcome! See <this link> to contribute! something friendlier.